lib/chef/resource/yum_repository.rb

```diff
@@ -34,12 +34,12 @@ class YumRepository < Chef::Resource
       # http://linux.die.net/man/5/yum.conf as well as
       # http://dnf.readthedocs.io/en/latest/conf_ref.html
       property :baseurl, [String, Array],
-        description: "URL to the directory where the Yum repository's 'repodata' directory lives. Can be an http://, https:// or a ftp:// URL. You can specify multiple URLs in one baseurl statement."
+        description: "URL to the directory where the Yum repository's 'repodata' directory lives. Can be an `http://`, `https://` or a `ftp://` URLs. You can specify multiple URLs in one baseurl statement."
```

Comment:
```suggestion
        description: "URL to the directory where the Yum repository's `repodata` directory lives. Can be an `http://`, `https://` or a `ftp://` URLs. You can specify multiple URLs in one `baseurl` statement."
```

Should these get backticks too?